seesaw/indices/roitracked/preprocessor.py

```diff
@@ -180,15 +179,15 @@ def preprocess_roi_dataset(
                     a['clip_feature_vector'] = clip_array
                     
                     clip_features += clip_array.tolist()
-                track_id = data['file_path'].split('/')[1]
+                track_id = data['file_path'].split('/')[0]
```

Comment:
the track id is meant to be a number assigned to multiple views of the same object (as matched by a tracker),
multiple tracks belong in the same video.

In the code above it seems like track_id is a string derived from some part of the file path, which is odd.